src/plugins/intel_cpu/src/node.h

```diff
@@ -214,7 +214,7 @@ class Node {
                 openvino::itt::handle<Tag<NodeType, 5>>(type_name + "::initOptimalPrimitiveDescriptor");
         }
 
-        openvino::itt::handle_t execute;
+        openvino::itt::handle_t execute{};
```

Comment:
Init non (do it in ctor) or apply to all.

Reply:
Done (moved to ctor)